### PR TITLE
Clean up pipeline docs a little

### PR DESCRIPTION
**Pull request by kenoir**
In preparation for writing up how to do a reindex a bit better, this cleans up some existing documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,33 @@
...
+#### Reindexing
 
-**Interested in other parts of the Wellome Collection digital platform
-works?**
+If the [internal_model](./common/internal_model) has been changed you will want to update the information stored by the pipeline to match that model.
 
-[Take a look at our Platform repo][platform repo]
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
 
-**Interested in how all of this works**
```
Comment (kenoir) [inline README.md:39]: This has all gone into the GitBook documentation.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,33 @@
...
-[Take look at our developer documentation][api developer docs] or
-[go straight to our API][api].
+#### Reindexing
 
-**Interested in other parts of the Wellome Collection digital platform
-works?**
+If the [internal_model](./common/internal_model) has been changed you will want to update the information stored by the pipeline to match that model.
 
-[Take a look at our Platform repo][platform repo]
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
```
Comment (jamesgorrie) [inline README.md:23]: Transformed makes sense, matched merged could use a link to the docs on that... which I don't htink exist.

File: docs/GITBOOK.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+![gitbook\_pr](https://user-images.githubusercontent.com/31692/70724110-e604ed00-1cf1-11ea-92b1-d7b2e4a61108.png)
+
+* Write up the review explaining what the docs are for
+* Follow up on any suggestions or changes
+* Once you've had a positive review, merge the changes, and delete the branch
+* Delete the variant in Gitbook  
+
+![gitbook\_delete\_variant](https://user-images.githubusercontent.com/31692/70724129-ec936480-1cf1-11ea-9101-3264053016c4.png)
+
```
Comment (jamesgorrie) [inline docs/GITBOOK.md:40]: This can be removed as no-one edits in Gitbook anymore.

File: docs/README.md [hunk 4]
```diff
@@ -1,42 +1,52 @@
...
+These are:
 
-![gitbook\_pr](https://user-images.githubusercontent.com/31692/70724110-e604ed00-1cf1-11ea-92b1-d7b2e4a61108.png)
+* Adapters: Syncing data from multiple external sources, enabling retrieving data performantly and at scale:
+  - [Sierra adapter](../sierra_adapter/README.md): [Sierra](https://www.iii.com/products/sierra-ils/) contains data on things in the library
+  - [Calm adapter](../calm_adapter/README.md): [Calm](https://www.axiell.com/uk/solutions/product/calm/) contains data on things in the archive
+  - [METS adapter](../mets_adapter/README.md): [METS](http://www.loc.gov/standards/mets/) data on digital assets from our workflow & archival storage systems. 
+* [Pipeline](./pipeline/README.md): Taking adapter data and putting it into our query index. We use [Elasticsearch](https://www.elastic.co/elasticsearch/) as our underlying search engine.
+* [API](https://github.com/wellcomecollection/catalogue-api/blob/main/README.md): The public APIs to query our catalogue data. The API services are stored in a different GitHub repository: https://github.com/wellcomecollection/catalogue-api
```
Comment (jamesgorrie) [inline docs/README.md:48]: Is all of this worth moving to `platform` and this speaks specifically about the pipeline?

**Review by jamesgorrie — APPROVED** [review]
Much neater!

File: README.md (hunk 2 above)
Comment (kenoir) [reply README.md:23]: I agree there should be some explanation of matching and merging in these docs, but I won't add that in this PR.

File: docs/README.md (hunk 4 above)
Comment (kenoir) [reply docs/README.md:48]: I think perhaps moving everything in docs into https://github.com/wellcomecollection/docs might be a good idea. 

Make the platform repo a placeholder pointing at the GitBook docs (and holding the ontologies as it does now).

Comment (jamesgorrie) [reply docs/README.md:48]: Yea, I like repos holding dev specific things, and more producty things in GB.

**Pull request merged**